File: UserManagement.Web/Models/CustomerDetailModel.cs
namespace UserManagement.Web.Models
{
    public class CustomerDetailModel
    {
        public int Id { get; set; }

        public string FirstName { get; set; }
        public string MiddleName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string AccountNumber { get; set; }
        public string BankName { get; set; }
        public string BankBranch { get; set; }
        public string Ifsc { get; set; }
        public decimal? Discount { get; set; }
        public AddressModel Address { get; set; }
        public int? AddressId { get; set; }
    }
}

File: UserManagement.Web/ClientApp/src/models/customer/customer.detail.model.ts
import { AddressModel } from '../address.model';

export class CustomerDetailModel {
    public id: number;
    public firstName: string;
    public middleName: string;
    public lastName: string;
    public email: string;
    public phone: string;
    public address: AddressModel;
    public discount: number;

    public accountNumber: string;
    public bankName: string;
    public branchName: string;
    public ifsc: string;
}
